backend/apps/web/routers/utils.py

```diff
@@ -96,8 +96,7 @@ async def download(
     file_name = parse_huggingface_url(url)
 
     if file_name:
-        os.makedirs("./uploads", exist_ok=True)
-        file_path = os.path.join("./uploads", f"{file_name}")
+        file_path = f"{UPLOAD_DIR}/{file_name}"
```

Comment:
I refactored code myself but next time if you make a PR, please stick to the f-string convention used! Thanks!

Reply:
fixed

BTW pathlib's / operator normalizes the path joins, for example using \ instead of / on Windows.